### ISSUE TITLE
How can I pass value to the input on the frontend?

### ISSUE THREAD

**Issue by lexbi** [roundhouse/FormBuilder-2-Craft-CMS#54]
Hi,

I've seen that you can pass an array for a value to getInputHtml: `public function getInputHtml($field, $value = [])` how would I write this on the front end? Twig is a bastard with arrays right so you have to pass it a "hash"?

Having no luck with: `{% set input = craft.formBuilder2.getInputHtml(field, {"value":craft.request.url}) %}`

Any ideas on what I can do to accomplish that?

Cheers

**Comment by lexbi** [comment]
This is possible using the method on this issue: https://github.com/roundhouse/FormBuilder-2-Craft-CMS/issues/25#issuecomment-227494483

Though I would still like to know how to pass data to the `$value` array (if that's possible)

**Comment by owldesign** [comment]
@lexbi do you have a use case for this? I'm not exactly sure why you would want to pass a value.

**Comment by lexbi** [comment]
I wanted to pass a url to an input like suggested in my first comment, not really required now as I've managed it with your other suggestion here: https://github.com/roundhouse/FormBuilder-2-Craft-CMS/issues/25#issuecomment-227502511

Curious how I would write it if I did want to do that. Could be useful if you wanted to pre-set a value dynamically? Though I guess this could still be accomplished with the method above.
